Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,27 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
 # final image needs kubectl, so we copy `oc` from cli image to use as kubectl.
-FROM openshift/origin-cli:latest as cli
+FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
 # the base image is the ansible-operator's origin images
-FROM quay.io/openshift/origin-ansible-operator:4.6
+FROM registry.svc.ci.openshift.org/ocp/4.6:ansible-operator
 
 USER root
-RUN set -x; INSTALL_PKGS="curl bash ca-certificates less which inotify-tools openssl" \
-    && yum clean all && rm -rf /var/cache/yum/* \
-    && yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm  \
-    && yum install --setopt=skip_missing_names_on_install=False -y \
-        $INSTALL_PKGS  \
-    && yum clean all \
-    && rm -rf /var/cache/yum
-
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
+RUN set -x;
+ARG INSTALL_PKGS="less python2-botocore python2-boto3 openssl tini"
```

Comment:
Move to yum install vs. pip install (pip cannot be used in OSBS/ART build) the following:
`botocore`,`boto3`
Note the following that previously were pip installed, but are now neither yum, nor pip, installed:
- `ansible` does not need to be installed, it's present from the base image.
- `openshift` does not need to be installed. 
- `cryptography` does not to be pip installed, yum install of `openssl` is sufficient.

Reply:
I think we still need to rethink deleting these packages and instead pin a particular version for them - that way, we don't run into the problem where the package we get from the base image does not differ between CI and ART again.

Reply:
For `ansible` we're pinning it presumably as the base image has semver specification and that's where it's coming from: `FROM registry.svc.ci.openshift.org/ocp/4.6:ansible-operator`.

For `openshift` it literally does not seem to be needed, as not installing it does not seem to harm anything. I think same logic as above for anisble might apply, because if some other package is grabbing it as a dependency, we only have two sources and they both have version explicitly specified (4.7:ansible-operator base image repos, or base-4.4.ocp.svc proxy to repos).

For `cryptography`, Ansible seems to actually need the openssl CLI client to create a random seed. Installing openssl takes care of this need and the python module does not seem needed. We could pin the openssl package to a point in time?